egs/hkust/s5/local/hkust_segment.py

```diff
@@ -12,4 +12,4 @@
       continue
     for j in seg_txt(blks[i]):
       out_line += " " + j
-  print out_line     
+  print out_line
```

Comment:
just noticed this -- please use 
from __future__ import print_function
and then

print(out_line)


that should be more opened towards python3 transition